lib/resource-serializer.js

```diff
@@ -1,9 +1,22 @@
 'use strict';
+var _ = require('lodash');
 var SerializerUtils = require('./serializer-utils');
 
 function ResourceSerializer(collectionName, record, opts) {
   var payload = { data: {} };
 
+  function getLinks(links) {
+    return _.mapValues(links, function (value) {
+      if (_.isFunction(value)) {
+        return value(records);
+      } else {
+        return value;
+      }
+    });
+  }
+
+  if (opts.topLevelLinks) { payload.links = getLinks(opts.topLevelLinks); }
+
```

Comment:
Any suggestion where to put this? I didn't want to create a new module for just this small function. The `getLinks` function in `serializer-utils.js` is also similar, so these could probably be merged somewhere.